packages/material-ui-unstyled/src/SliderUnstyled/SliderValueLabelUnstyled.js

```diff
@@ -30,7 +30,7 @@ function SliderValueLabelUnstyled(props) {
     {
       className: clsx(children.props.className),
     },
-    <Root className={clsx(classes.offset, className)} theme={theme}>
+    <Root className={clsx(classes.offset, className)} theme={theme} aria-hidden>
```

Comment:
Since it usually just displays the same content as `aria-valuenow` or `aria-valuetext`, not only was it repeated content, but it was confusing to the user since in their mind they would have left the slider input and wouldn't understand where they were at now. It is also a little annoying when they are in a form with multi inputs and they have to swipe twice to get to the next input.